### ISSUE TITLE
missing publication about differential expression study

### ISSUE THREAD

**Issue by nikiforosk** [opentargets/data-providers-docs#21]
PRDM1 - asthma
https://www.targetvalidation.org/evidence/ENSG00000057657/EFO_0000270?view=sec:rna_expression

There is one RNA expression study supporting this association. The Publications column is empty.

However, if you click on the link under "Experimental overview" you go to [this page](http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-61141/) which includes a citation for the study: http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0118286

Shouldn't this paper be reported under the Publications column of the Evidence page?

Notice that this study supports the association of many genes with asthma (so it will probably be included in many Evidence pages):
http://www.ebi.ac.uk/gxa/experiments/E-GEOD-61141

**Comment by nikiforosk** [comment]
Can we ask them to check if there are cases like that where the supporting publication is missing?

**Comment by ckongEbi** [comment]
@nikiforosk looking at other examples, raw data & the json schema , it looks to me that literature references was never a mandatory field. I suspect the "Publication" column is always empty for RNA expression. 

https://targetvalidation.org/evidence/ENSG00000100368/EFO_0003767?view=sec:rna_expression

**Comment by nikiforosk** [comment]
We used to show publication links at some point. I attach a screenshot of BRAF in neoplasm:
![03-04-rna-screen shot 2015-06-17 at 11 49 15 2](https://cloud.githubusercontent.com/assets/20280139/25696699/0e5e5d26-30b0-11e7-9e8c-24f2bc060a04.png)

If you look at the same page now, there are no publications:
https://www.targetvalidation.org/evidence/ENSG00000157764/EFO_0000616?view=sec:rna_expression

So you're right they don't seem to be providing Publication links. We want them though. 

So in the example above (PRDM1 in asthma):
https://www.targetvalidation.org/evidence/ENSG00000057657/EFO_0000270?view=sec:rna_expression

we should be linking to this publication:
http://europepmc.org/abstract/MED/25706956

as mentioned in this page from the Atlas
http://www.ebi.ac.uk/gxa/experiments/E-GEOD-61141
and from Array express:
http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-61141/

(Notice that this link will populate many Publication links since the association of asthma with many targets stem from this particular study).

I think we want Publications to remain optional (when there is no publication for a datapoint) but we want the link to the Publications when they exist. Can you please raise this with the Atlas curators?

**Comment by iandunham** [comment]
Have you checked the JSON for the examples that Niki highlighted? Are the publications there in the json. You can also check the previous data dumps. They definitely should be some.

If they are there in the current json, maybe a schema change has happened?

**Comment by deniseOme** [comment]
IWith the API evidence call, I can see the GEOD (61141) but no Europe PMC publication:

https://www.targetvalidation.org/api/latest/public/evidence/filter?target=ENSG00000057657&disease=EFO_0000270&datatype=rna_expression

The other thing that is strange from the GUI is that we link to Array Express, when I thought we should like to Expression Atlas...

Linking to 
http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-61141/

but perhaps the one below would be more useful to the user?
http://www.ebi.ac.uk/gxa/experiments/E-GEOD-61141?accessKey=&serializedFilterFactors=&queryFactorType=&rootContext=&heatmapMatrixSize=50&displayLevels=false&displayGeneDistribution=false&geneQuery=%5B%7B%22value%22%3A%22PRDM1%22%2C%22category%22%3A%22symbol%22%7D%5D&regulation=UP_DOWN&_queryFactorValues=1&specific=true&_specific=on&cutoff=0.05&foldChangeCutOff=1.0

I've discussed this with Laura and she also found the linking to Array Express not that appropriate either.

**Comment by ckongEbi** [comment]
@iandunham publication is not in the JSON, will have to dig into the schema history to see when it was change and why. In any case, we will raised this to them to have it added.

@deniseOme we can speak to Laura tomorrow on the atlas links

**Comment by iandunham** [comment]
@ckongEbi check whether the publications are in the submitted JSON, and if not go back to Atlas to find out where the publications have gone.

@deniseOme The purpose of that link is to give details of the experiment and data isn't it, rather than the specific data point? That's why it goes back to Array Express. If we change the link the column label needs to be changed, but I don't think we necessarily change the link

**Comment by ckongEbi** [comment]
@iandunham publication are not in the original submitted JSON (at least for this example), we will find out why

**Comment by deniseOme** [comment]
@iandunham, I expected to land on Expression Atlas (EA), as EA is the source for our evidence, not AE. I did fail to pay attention to the name of that column as 'Experiment overview and data'.

The column named 'Evidence source' does not get linked out to the evidence source, i.e. EA. Wouldn't it be useful to do so? As I user I'd like to get a direct link to EA for the differential expression of PRMD1. We can get from EA from the current AE link but it takes one to a generic page  in EA that includes all genes that are differentially expressed rather than the gene of interest (e.g. PRMD1).

**Comment by iandunham** [comment]
We do link out to the evidence source at the top of the page: "Source: Expression Atlas". 
The choices about where to link to in these tables had a lot of background work originally by @nikiforosk so we shouldn't change them unless we really think it's right. I think the original thinking was that linking out to the single gene expression was not necessarily useful, because the colouring is dysfunctional in this page - it is auto generated and is always either strong red or blue, because the scaling has no context.  That colouring doesn't reflect the overall rank of effect in the full experiment. There is also an issue about linking to the overall summary page because your gene may not b e in the top 50 strongest effects, so won't be visible.

**Comment by deniseOme** [comment]
Sure, thank you. The link "Source" Expression Atlas" takes us to our Data Sources page in our Platform, rather than the Expression Atlas page itself. 

As we want to give users the provenance of the data for their own judgment, I think it is important to link to the actual data point we used as evidence. 

We seem to link to actual data points in some cases but not others. So the evidence pages seem not to be consistent depending on the data type we look at. I will give a few examples below but will leave it up to @nikiforosk and @emepyc to revise this if needed be, in which case this issue may be created in another repo.

A) Link out to Clinical Trials i.e.
https://www.targetvalidation.org/evidence/ENSG00000142192/EFO_0005772?view=sec:known_drugs

e.g. SOLANEZUMAB phase III (etc) linked to NCT00905372 on https://clinicaltrials.gov/search?id=%22NCT00905372%22

B) Link out to the ClinVar variant for somatic mutation i.e.
https://www.targetvalidation.org/evidence/ENSG00000139618/EFO_0001075?view=sec:somatic_mutations

e.g. stop gained linked to https://www.ncbi.nlm.nih.gov/clinvar/RCV000113766/

C) Link out to JAX for mouse models i.e.
https://www.targetvalidation.org/evidence/ENSG00000139618/EFO_0000508?view=sec:animal_models

e.g. knockout BRCA2 linked to http://www.informatics.jax.org/allele/MGI:2387337

D) link out to GWAS for data on the SNPs:
https://www.targetvalidation.org/evidence/ENSG00000139618/EFO_0000616?view=sec:genetic_associations

e.g. rs11571833 linked to https://www.ebi.ac.uk/gwas/search?query=rs11571833

**Comment by deniseOme** [comment]
Discussions with @iandunham & @nikiforosk: we do link to Expression Atlas already. Click on 'unknown' below to get to the EA page for the gene of interest i.e. PRDM1:

http://www.ebi.ac.uk/gxa/experiments/E-GEOD-61141?geneQuery=ENSG00000057657

**Comment by ckongEbi** [comment]
@iandunham @nikiforosk atlas will look at the publication fix
@deniseOme we will keep the link to AX as it is,  can you move the other link out issues to the more relevant repo, webapp maybe?

**Comment by ckongEbi** [comment]
@iandunham @nikiforosk this is now fix in `17.06` if not please reopen issue
